pkg/cloudprovider/aws/securitygroups.go

```diff
@@ -46,11 +45,6 @@ func (s *SecurityGroupProvider) Get(ctx context.Context, constraints *v1alpha1.C
 	if err != nil {
 		return nil, err
 	}
-	// This hack works around
-	// https://github.com/kubernetes-sigs/aws-load-balancer-controller/issues/2367
-	// The LoadBalancer Controller expects a single security group with the
-	// cluster tag, but provisioning tools like eksctl and kops create multiple.
-	securityGroups = s.filterClusterTaggedGroups(ctx, securityGroups)
```

Comment:
Looking at line 63:
`if value == "*" {`
I think this is the default behavior of the API, since it supports wildcards. We may be able to simplify.

Reply:
Make sure we point to  https://github.com/kubernetes-sigs/aws-load-balancer-controller/issues/2367 in the docs

Reply:
Nice!

I didn't find any mention of it in the documentation but it worked when I tried it out.

```
aws ec2 describe-security-groups --filters Name=vpc-id,Values=vpc-123 Name=tag-key,Values=kubernetes.io/cluster/myCluster

and 

aws ec2 describe-security-groups --filters Name=vpc-id,Values=vpc-123 Name=tag:kubernetes.io/cluster/myCluster,Values="*"
```

gave me identical results, so using just the latter.